### ISSUE TITLE
Adding new loss function for multi-label classification

### ISSUE THREAD

**Issue by theravadaz** [open-mmlab/mmaction2#1805]
Hi :) I'm currently using MMAction2 for a multi-label classification task, with a BCE with logits loss function. However, I want to write my own loss function (a variant of BCE loss). I think I understand how to implement a new loss function in MMAction2, but I'm really confused how to do this for a multi-label task. For a given datapoint, I want to calculate the loss for each individual class. Does this just mean that the loss returned by my function is a vector, rather than a number?

Thanks for the help!

**Comment by hukkai** [comment]
After you  calculate the loss for each individual class, you need to average or sum theses losses, right? So I think it is still a number.

**Comment by hukkai** [comment]
If you have any further questions, feel free to re-open the issue. Thanks!
